benches/payout-stakers/src/main.rs

```diff
@@ -33,53 +32,68 @@ fn main() -> Result<(), anyhow::Error> {
     info!("Starting benchmark with config ");
 
     let connection = create_connection(address).set_signer(sudoer);
-
-    let nominator_accounts = generate_nominator_accounts_with_minimal_bond(&connection);
     let validators = set_validators(address);
-    let nominee = &validators[0];
-    nominate_validator_0(&connection, nominator_accounts.clone(), nominee);
-    wait_for_10_eras(address, &connection, nominee, nominator_accounts)?;
+    let mut validators_and_its_nominators = Vec::new();
+    validators
```

Comment:
nit: validators_and_its_nominators can be created from validators as a transformation rather than a mutable push. 

not nit: Extract this to function for example called `test_validators_and_its_nominators()` this might be useful in the future.